### ISSUE TITLE
writing a permenant note on the cards using whiteboard tool or any other one.

### ISSUE THREAD

**Issue by hssm** [ankidroid/Anki-Android#2897]
Originally reported on Google Code with ID 1996

```
hi there.at first i want to thanks for your great app.
i was wondering if there is any way to have the notes which we write on the screen,using
the whiteboard tool or any other one,to be saved on the cards as a part of them ,like
when you are reading an example sentence in the answer and you wanna draw a circle
around a specific word or write a note in a specific field or on the card itself ,instead
of typing it.in fact the very unique ability that a paper flashcard gives you is that
you can write your own notes on it,which is a very useful way to better memorizing.

Is this feature available on similar apps (Anki Desktop, AnkiMobile,
AnyMemo, etc) ? If yes, it could help to include a screenshot or link to a
page explaining the feature.
 i don't think so.

thanks a lot again!
```

Reported by `bigfazelcity` on 2014-02-15 20:38:35

**Comment by onerror** [comment]
@hssm This is a very important feature. Where is it merged to? 2064 is not about this issue

**Comment by timrae** [comment]
Please stop spamming our closed issues and read my comment in #3497
